plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -258,7 +258,7 @@
 
     private String _updateHostPasswdPath;
 
-    private int _dom0MinMem;
+    private long _dom0MinMem;
```

Comment:
I think we need to initialize to a reasonable default or 0L. Not sure if it is possible by installation but otherwise the properties file might be corrupted or empty.